examples/custom-universal-routing/next.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  customRoutes: [{
+    test: /^\/$/,
+    routeTo: '/foo'
+  }, {
+    test: /^\/about(\/?)$/,
```

Comment:
Use [path-to-regexp](https://github.com/pillarjs/path-to-regexp) for express like route URLs.